src/main/java/io/stargate/sgv2/jsonapi/service/schema/collections/CollectionSchemaObject.java

```diff
@@ -107,115 +113,25 @@ public IndexingProjector indexingProjector() {
-    }
-
-    public static IndexingConfig fromJson(JsonNode jsonNode) {
-      Set<String> allowed = new HashSet<>();
-      Set<String> denied = new HashSet<>();
-      if (jsonNode.has("allow")) {
-        jsonNode.get("allow").forEach(node -> allowed.add(node.asText()));
-      }
-      if (jsonNode.has("deny")) {
-        jsonNode.get("deny").forEach(node -> denied.add(node.asText()));
-      }
-      return new IndexingConfig(allowed, denied);
-    }
-
-    // Need to override to prevent comparison of the supplier
-    @Override
-    public boolean equals(Object o) {
-      if (this == o) return true;
-      if (o instanceof IndexingConfig other) {
-        return Objects.equals(this.allowed, other.allowed)
-            && Objects.equals(this.denied, other.denied);
-      }
-      return false;
-    }
-  }
-
-  /**
-   * The similarity function used for the vector index. This is only applicable if the vector index
-   * is enabled.
-   */
-  public enum SimilarityFunction {
-    COSINE,
-    EUCLIDEAN,
-    DOT_PRODUCT,
-    UNDEFINED;
-
-    public static SimilarityFunction fromString(String similarityFunction) {
-      if (similarityFunction == null) return UNDEFINED;
-      return switch (similarityFunction.toLowerCase()) {
-        case "cosine" -> COSINE;
-        case "euclidean" -> EUCLIDEAN;
-        case "dot_product" -> DOT_PRODUCT;
-        default ->
-            throw ErrorCodeV1.VECTOR_SEARCH_INVALID_FUNCTION_NAME.toApiException(
-                "'%s'", similarityFunction);
-      };
-    }
-  }
-
-  /** Collection Id Type enum, UNDEFINED represents unwrapped id */
-  public enum IdType {
-    OBJECT_ID,
-    UUID,
-    UUID_V6,
-    UUID_V7,
-    UNDEFINED;
-
-    public static IdType fromString(String idType) {
-      if (idType == null) return UNDEFINED;
-      return switch (idType) {
-        case "objectId" -> OBJECT_ID;
-        case "uuid" -> UUID;
-        case "uuidv6" -> UUID_V6;
-        case "uuidv7" -> UUID_V7;
-        case "" -> UNDEFINED;
-        default -> throw ErrorCodeV1.INVALID_ID_TYPE.toApiException(idType);
-      };
-    }
-
-    public String toString() {
-      return switch (this) {
-        case OBJECT_ID -> "objectId";
-        case UUID -> "uuid";
-        case UUID_V6 -> "uuidv6";
-        case UUID_V7 -> "uuidv7";
-        case UNDEFINED -> "";
-      };
-    }
-  }
-
-  public enum AuthenticationType {
-    NONE,
-    HEADER,
-    SHARED_SECRET,
-    UNDEFINED;
-
-    public static AuthenticationType fromString(String authenticationType) {
-      if (authenticationType == null) return UNDEFINED;
-      return switch (authenticationType.toLowerCase()) {
-        case "none" -> NONE;
-        case "header" -> HEADER;
-        case "shared_secret" -> SHARED_SECRET;
-        default ->
-            throw ErrorCodeV1.VECTORIZE_INVALID_AUTHENTICATION_TYPE.toApiException(
-                "'%s'", authenticationType);
-      };
-    }
-  }
+  // TODO: AARON COMMENTED OUT TO SEE IF IT IS USED
```

Comment:
Can we remove this code if not used?

Reply:
tracking in https://github.com/stargate/data-api/issues/1458